CodeStandard.md

```diff
@@ -0,0 +1,51 @@
+# Code Standard
+
+## When to document
+
+Rule of thumb - Continuous Documentation
+
+BUT! Visual studio can do a lot of it for you!
+
+You start by typing `///` above where you want to put the code comment.
+
+So if you add code comments in when you about to go into review, that's A-OK. I wont know ;) but git-history will.
+
+## Other Notes
+
+1. Rule of thumb - Continuous Documentation.
+2. Be sure to always keep a space between props and functions.
+3. In function comments are with two slashes. `//`
+4. No Magic Numbers: Values that don't have an obvious meaning
+
+## Example Class
+
+```C#
+/// <summary>
+/// Add a summary of the Class
+/// </summary>
+public class ClassName : ParentClassName
+{
+
```

Comment:
Is this empty line needed/part of some convention?

Reply:
No that might be just my habit.